src/utils/HttpError.ts

```diff
@@ -0,0 +1,11 @@
+class HttpError extends Error {
```

Comment:
That could be our  BaseError and other errors will extends it. TaskNotFoundError, ValidationError etc.

Reply:
fixed e1098e4418555c67b50397f454c93ff0a4312a66